integration/config/configtx.yaml

```diff
@@ -259,15 +259,7 @@ Orderer: &OrdererDefaults
 
     # Orderer Type: The orderer implementation to start.
     # Available types are "solo" and "kafka".
-    OrdererType: etcdraft
-    EtcdRaft:
-        Consenters:
-        - Host: orderer.example.com
-          Port: 7059
-          ClientTLSCert: organizations/ordererOrganizations/example.com/orderers/orderer.example.com/tls/server.crt
-          ServerTLSCert: organizations/ordererOrganizations/example.com/orderers/orderer.example.com/tls/server.crt
-    Addresses:
-        - orderer.example.com:7050
+    OrdererType: solo
```

Comment:
this looks wrong

Reply:
it looks wrong but it actually isn't (i stumbled also over this): We select a profile (`SampleDevModeEtcdRaft`) where orderer type is correctly set to raft, so this is fine ...

Reply:
ok got it - this defines &OrdererDefaults and sets OrdererType to solo. Changing it here to EtcdRaft would cause the SampleDevModeSolo profile not to work anymore.